experimental/datetime_bag_v2/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "datetime_bag_v2"
```

Comment:
@sffc @nordzilla ought this be a separate crate or just an experimental module in DTF?

Reply:
Made a new crate based on [this](https://github.com/unicode-org/icu4x/issues/1318#issuecomment-1040752383) comment, but that was several months ago.

Reply:
Ah, okay. I wasn't super plugged in to that part of that decision so I'll defer to the folks who are

Reply:
I stand by my suggestion from February:

> The lion's share of the code in the datetime crate is built on assumptions of the old data model, and many of the pieces independent of the data model have been moved to the calendar crate. I therefore believe the best place to start working on this is in a new experimental crate. At the end, once things are working, we will reconcile the two.

Reply:
Works for me!